src/DotNetEnv/LoadOptions.cs

```diff
@@ -4,47 +4,54 @@ namespace DotNetEnv
+        public bool ClobberExistingVars { get; set; }
+        public bool OnlyExactPath { get; set; }
 
         public LoadOptions(
             bool setEnvVars = true,
             bool clobberExistingVars = true,
             bool onlyExactPath = true
-        ) {
+        )
+        {
             SetEnvVars = setEnvVars;
             ClobberExistingVars = clobberExistingVars;
             OnlyExactPath = onlyExactPath;
         }
 
-        public LoadOptions(
-            LoadOptions old,
-            bool? setEnvVars = null,
-            bool? clobberExistingVars = null,
-            bool? onlyExactPath = null
-        ) {
-            SetEnvVars = setEnvVars ?? old.SetEnvVars;
-            ClobberExistingVars = clobberExistingVars ?? old.ClobberExistingVars;
-            OnlyExactPath = onlyExactPath ?? old.OnlyExactPath;
-        }
+        public static LoadOptions DEFAULT => new LoadOptions();
+
+        public static LoadOptions NoEnvVars(LoadOptions options = null) =>
+            (options ?? DEFAULT).NoEnvVars();
+
+        public static LoadOptions NoClobber(LoadOptions options = null) =>
+            (options ?? DEFAULT).NoClobber();
 
-        public static LoadOptions NoEnvVars (LoadOptions options = null) =>
-            options == null ? DEFAULT.NoEnvVars() : options.NoEnvVars();
+        public static LoadOptions TraversePath(LoadOptions options = null) =>
+            (options ?? DEFAULT).TraversePath();
+    }
 
-        public static LoadOptions NoClobber (LoadOptions options = null) =>
-            options == null ? DEFAULT.NoClobber() : options.NoClobber();
+    public static class LoadOptionExtensions
+    {
+        public static LoadOptions NoEnvVars(this LoadOptions options)
+        {
+            options.SetEnvVars = false;
+            return options;
+        }
 
-        public static LoadOptions TraversePath (LoadOptions options = null) =>
-            options == null ? DEFAULT.TraversePath() : options.TraversePath();
+        public static LoadOptions NoClobber(this LoadOptions options)
+        {
+            options.ClobberExistingVars = false;
+            return options;
+        }
 
-        public LoadOptions NoEnvVars () => new LoadOptions(this, setEnvVars: false);
-        public LoadOptions NoClobber () => new LoadOptions(this, clobberExistingVars: false);
-        public LoadOptions TraversePath () => new LoadOptions(this, onlyExactPath: false);
+        public static LoadOptions TraversePath(this LoadOptions options)
+        {
+            options.OnlyExactPath = false;
+            return options;
+        }
 
-        public IEnumerable<KeyValuePair<string, string>> Load (string path = null) => Env.Load(path, this);
-        public IEnumerable<KeyValuePair<string, string>> LoadMulti (string[] paths) => Env.LoadMulti(paths, this);
+        public static IEnumerable<KeyValuePair<string, string>> Load(this LoadOptions options, string path = null) => Env.Load(path, options);
```

Comment:
Why are you making these extension methods when they could more easily live on the actual object, as they did before your changes?

Reply:
Again some personal preference ;)

If we did not need to maintain best possible backwards-compatibility, I would say: let's remove all the methods from LoadOptions, leaving only Props.
Since it is a ConfigurationObject, it should be as pure as possible, just a POCO.

Me personally wouldn't even allow to call `Load()` directly on the options.
There is no real benefit, except for personal preferences. Why not calling `Env.LoadMulti()` with parameter Options, the method already exists and is only wrapped?
Imho it feels wrong to call Load directly on the options, because the LoadOptions don't need to know anything about Env.Load etc.
That's also the reason for moving it to an extension, it's a bit like instead of removing it completely.
The LoadOptions don't need to have any dependency to the class `Env`.

Sure, nothing happens due to this Dependency, but I really like to keep my dependencies clean.
That allows to separate things better later on if needed.